libc-test/build.rs

```diff
@@ -3186,7 +3186,7 @@ fn test_neutrino(target: &str) {
 
     cfg.skip_type(move |ty| {
         match ty {
-            // FIXME: `sighandler_t` type is incorrect, see:
+            // FIXME(neutrino): `sighandler_t` type is incorrect, see:
             // https://github.com/rust-lang/libc/issues/1359
```

Comment:
This one is kind of a special case. Based on the link below, `FIXME(sighandler)` is probably better since this same thing needs to be updated in a few places.